src/test/java/org/folio/marc4ld/mapper/Marc2BibframeMapperIT.java

```diff
@@ -1114,4 +1114,30 @@ private void validateId(Resource resource) {
     var expectedId = hashService.hash(resource);
     assertThat(resource.getId()).isEqualTo(expectedId);
   }
+
+
+  @Deprecated(forRemoval = true, since = "use test utils")
```

Comment:
Created a new ticket [MODLD-391](https://folio-org.atlassian.net/browse/MODLD-391). Let us add it as a `TODO` comment in this file.